src/screens/General/Dashboard.js

```diff
@@ -349,10 +350,10 @@ class Dashboard extends Component {
 		} = styles;
 
 		let buttonLinks = [
-			["https://shpeucf2018-2019.slack.com/", "slack"],
-			["https://www.facebook.com/shpeucfchapter/", "facebook"],
+			[`https://www.slack.com/channel?team=${slackInfo.team}&id=${slackInfo.channel.general}`, "slack"],
```

Comment:
```suggestion
					let buttonLinks = [{
					url: `https://www.slack.com/channel?team=${slackInfo.team}&id=${slackInfo.channel.general}`, 
					appName: "slack"
					},
					...
					{
					url: "https://www.instagram.com/shpeucf",
					appName: "instagram"
					}
				]
```
I think something like this would produce more readable code.

Reply:
Also Maybe call it socialMediaLinks instead of buttonLinks or something else that explains what exactly is being held in the variable.